src/PageContent/Calendar/Dialog.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import './Dialog.css';
+import Header from './Header';
+import DialogContent from './DialogContent';
+
+class Dialog extends Component {
+    render() {
+        return (
+            <dialog className="event-dialog" open>
+                <Header
+                    eventType={this.props.delivery.eventType}
+                    closeDialog={this.props.closeDialog}
+                    date={this.props.delivery.date}
+                    startTime={this.props.delivery.startTime}
+                    endTime={this.props.delivery.endTime}
+                    futureEvent={this.props.futureEvent}
+                    accountType={this.props.accountType}
+                />
+                <DialogContent
+                    accountType={this.props.accountType}
+                    futureEvent={this.props.futureEvent}
+                    delivererGroup={this.props.delivery.delivererGroup.name}
```

Comment:
if you're passing in `this.props.delivery` below, i don't think you need to pass `this.props.delivery.delivererGroup.name` down separately